.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Issue/Task Number:
+
+# Overview
+
+Describe what your Pull Request is about in a few sentences. This is a global description that should tell people what your PR is all about. Don't go too much into details here.
```

Comment:
What do you think about chopping this down to just the first sentence? The rest of the description seems to be just rephrasing the same instructions using different words. 😃 

> Describe what your Pull Request is about in a few sentences

Reply:
@chrishunt should have used the new 'suggest change' feature ;)

Reply:
Just saw that! I may try that next time 😄